### ISSUE TITLE
simplify toggling of exit node

### ISSUE THREAD

**Issue by DeedleFake** [DeedleFake/trayscale#67]
Currently, enabling an exit node involves opening the window, finding the exit node in the list, and turning it on or off. However, it is most common, at least for me, to reuse the same exit node repeatedly, not switch them a bunch. So, a new feature:

* Remember which node was last used as an exit node.
* Add a toggle to the tray icon's menu and probably somewhere in-app to enable and disable that node as the exit node.

**Comment by mfat** [comment]
Another vote for this. Even a cascading menu to activate desired exit node will be great.

**Comment by jordan-bravo** [comment]
> Another vote for this. Even a cascading menu to activate desired exit node will be great.

Could you post a screenshot to an example of what you have in mind?

**Comment by slagiewka** [comment]
By cascading, I think he might mean nested, e.g. 

![image](https://github.com/user-attachments/assets/675d7211-3741-44da-92a2-8516e1226fcd)

**Comment by slagiewka** [comment]
Of course, there are multiple approaches to this. Official client splits checking network devices and choosing exit nodes to be two different things:
<img width="643" alt="Screenshot 2024-07-15 at 21 34 37" src="https://github.com/user-attachments/assets/2579e3af-6aa7-4f55-8c28-a230ddc8667d">

**Comment by theepicflyer** [comment]
Old issue but would like this as a feature as well. Similarly, I toggle between exit node on and off a lot, using the same exit node most of the time.
